styles/main.css

```diff
@@ -0,0 +1,526 @@
+body
+{
+    font-family: Arial;
+
+    margin: 0;
+    padding: 0;
+}
+
+.container
+{
+    padding: 0 10%;
+}
+
+header
+{
+    height: 50vh;
+
```

Comment:
No empty lines in the middle of a css selector please.
Also, be careful with using vh, make sure it looks good with shorter window heights. Your site doesn't look quite right if you have a short window.